perf/bchar_perf.ml

```diff
@@ -0,0 +1,22 @@
+open Perf
+
+let _ = 
+  runPerfTest "getDigit" (fun _ ->
+      let c = Bchar.getDigit '1' in
+      match c with
+      | Some _ -> ()
+      | None -> ()
+    );
+  runPerfTest "getDigitOrRaise" (fun _ -> 
+      let _ = (try 
+                 Bchar.getDigitOrRaise '1'
+               with
+               | Failure _ -> 1) in ();
+    );
+  runPerfTest "getDigitOrRaise Fail" (fun _ ->
+      let _ = (try 
+                 Bchar.getDigitOrRaise 'a' 
+               with 
+               | Failure _ -> 1
+              ) in (); 
```

Comment:
The `try...with` clause typed as integer so I have to wrap it around `let _ = (...) in ()`. Do you know how could I get around it?

I will try to reformat it in my next PR.